src/external/libtevent.m4

```diff
@@ -13,3 +13,32 @@ AS_IF([test x"$found_tevent" != xyes],
                       [-L$sss_extra_libdir -ltalloc])],
         [AC_MSG_ERROR([tevent header files are not installed])])]
 )
+
+SAVE_CFLAGS=$CFLAGS
+SAVE_LIBS=$LIBS
+CFLAGS="$CFLAGS $TEVENT_CFLAGS"
+LIBS="$LIBS $TEVENT_LIBS"
+build_chain_id=yes
+AC_CHECK_FUNCS([tevent_set_trace_fd_callback \
```

Comment:
Can this be replaced by just requiring libtevent > 0.11.0?

Reply:
Explicit checks are safer.
You never know what versioning approach could take other distribution...